wrappers/kn/src/nativeMain/kotlin/zxingcpp/BarcodeReader.kt

```diff
@@ -0,0 +1,19 @@
+package zxingcpp
+
+import kotlinx.cinterop.ExperimentalForeignApi
+import zxingcpp.cinterop.zxing_ReadBarcode
+import zxingcpp.cinterop.zxing_ReadBarcodes
+
+@OptIn(ExperimentalForeignApi::class, ExperimentalStdlibApi::class)
+class BarcodeReader {
+	fun readBarcode(imageView: ImageView, options: ReaderOptions? = null): Barcode? =
```

Comment:
I'd prefer if we used the same API here as in the Android, Rust, .NET rapper. Meaning, there is only one `read()` function and it calls the plural version.

Also I'd like to pass the Options parameter to the `BarcodeReader` constructor. Or maybe even go as far as making the reader class a subclass of the Options class. That is what I did with the Rust wrapper. Admittedly, this should be the same in the Android wrapper, to make it consistent. That is not set in stone, yet.

Reply:
fixed